app/models/DraftAvailability.scala

```diff
@@ -22,8 +22,8 @@ object DraftAvailability {
 
   def apply(departureSummaries: Option[DeparturesSummary]): DraftAvailability = departureSummaries match {
     case Some(value) if value.userAnswers.nonEmpty => NonEmpty
-    case Some(_)                                   => Empty
-    case None                                      => Unavailable
+    case Some(value) if value.userAnswers.isEmpty  => Empty
```

Comment:
Very minor but this case can only be hit if `value.userAnswers.isEmpty` is true so technically `if value.userAnswers.isEmpty` isn't needed

Reply:
@jackheslop96  done... please re -review